tests/test_wave_01.py

```diff
@@ -142,13 +141,15 @@ def test_moves_movie_from_watchlist_to_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
-    
-    raise Exception("Test needs to be completed.")
+    assert updated_data["watched"][1]["title"] == MOVIE_TITLE_1
```

Comment:
Nice assert! We can consider making it a bit more robust by checking that the entire movie dictionary matches.